### PR TITLE
Sort and Dedup Tessie 

### PR DESCRIPTION
**Pull request by Bre77**
## Proposed change

Somehow my merge conflicts have duplicated sections. This PR sorts headings alphabetically and removes the duplicates.

Targets next branch since Tessie doesnt exist in current.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [ ] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [ ] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/tessie.markdown [hunk 1]
```diff
@@ -82,31 +82,35 @@ The integration will create button entities to control various aspects of the ve
 - Keyless driving
 - Play fart
 
+### Climate
+
+The integration will create a climate entity to control the vehicles climate control system. This entity can:
```
Comment (frenck) [inline source/_integrations/tessie.markdown]:
```suggestion
The integration will create a climate entity to control the vehicle's climate control system. This entity can:
```

File: source/_integrations/tessie.markdown [hunk 2]
```diff
@@ -82,31 +82,35 @@ The integration will create button entities to control various aspects of the ve
...
 
-The integration will create device tracker entities for the vehicle's location and navigation destination.
+- Charge current
+- Charge limit
+- Speed limit
 
 ### Select
 
-The integration will create a select entity to control each seat heater. It allows you to set each seat heater to Off, Low, Medium, or High.
+The integration will create a select entity to control each of the seat heaters. It allows you to set each seat heater to Off, Low, Medium, or High.
```
Comment (frenck) [inline source/_integrations/tessie.markdown:132]:
```suggestion
The integration will create a select entity to control each seat heater. It allows you to set each seat heater to Off, Low, Medium, or High.
```

**Comment by Bre77** [comment]
Merge conflict fixed, Frenck feel free to change as much as you want.

**Review by frenck — APPROVED** [review]
Thanks, @Bre77 👍 

../Frenck

**Pull request merged**